### DISCUSSION TITLE
qianyan

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#937] [Announcements]
# qianyan

Neko首次诞生于2022.2.22，那是一个平凡的日子。 尽管自诞生以来，Neko历经了多次洗礼，也走过了各种坎坎坷坷…… 但我们永不放弃，无怨无悔。 如今Neko历经了两个完整的春夏秋冬，Neko也成长到了1.14版本。 希望以后的道路，能够让Neko看到耀眼的光芒。

https://docs.nekodayo.xyz/qianyan

**Comment by JQ-28** [comment]
😭

**Reply by 3553213** [reply]
jq抱抱

**Reply by JQ-28** [reply]
抱抱
